modules/materials/materials.model.js

```diff
@@ -0,0 +1,18 @@
+const materialType = ` 
+    _id: ID!
+    name: [Language!]
+    description: [Language!]
+    colors: [Color!] 
+    available: Boolean!
+    additionalPrice: Float!
```

Comment:
Please, note that when structural changes need to be made into the working DB down the line, every non-nullable field is more of an issue than a nullable one

Reply:
fixed